LeaderboardBackend/openapi.json

```diff
@@ -1429,12 +1376,22 @@
         "additionalProperties": false,
         "description": "This request object is sent when a `User` is attempting to register."
       },
-      "Run": {
+      "RunStatus": {
+        "enum": [
+          0,
+          1,
+          2,
+          3,
+          4
+        ],
+        "type": "integer",
+        "description": "The status of the `Run`.<br />\r\n    - 0: Created<br />\r\n    - 1: Submitted<br />\r\n    - 2: Pending<br />\r\n    - 3: Approved<br />\r\n    - 4: Rejected",
+        "format": "int32"
+      },
```

Comment:
Pending conclusion in Discord, but we may need to make sure this gets removed, as run status isn't part of the current DB spec.

Reply:
Yeah we should remove this. We don't have a run approval mechanism for now.